### PR TITLE
Ensure that the pcntl_signal_dispatch and pcntl_signal functions aren…

### PR DESCRIPTION
**Pull request by danog**
…'t disabled before enabling signal handling

### CODE REVIEW

**Review by kelunik — APPROVED** [review]

**Comment by kelunik** [comment]
@danog Could you please add a test? Might be best tested in a PHPT test.

**Comment by danog** [comment]
Found an easier way :)

**Review by kelunik — APPROVED** [review]

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -14,6 +14,11 @@ jobs:
           - operating-system: 'ubuntu-latest'
             php-version: '8.1'
             composer-flags: '--ignore-platform-req=php'
+            phpunit-flags: '-d disable_functions=pcntl_signal_dispatch'
+          - operating-system: 'ubuntu-latest'
+            php-version: '8.1'
+            composer-flags: '--ignore-platform-req=php'
+            phpunit-flags: '-d disable_functions=pcntl_signal_dispatch'
```
Comment (kelunik) [inline .github/workflows/ci.yml:22]: `pcntl_signal_dispatch` is now disabled in both runs.

**Review by kelunik — CHANGES_REQUESTED** [review]

File: .github/workflows/ci.yml (hunk 1 above)
Comment (kelunik) [reply .github/workflows/ci.yml:22]: Also, could you please add the flags so they show up in the job name?

Comment (danog) [reply .github/workflows/ci.yml:22]: Whoops, fixed :)

**Review by kelunik — APPROVED** [review]

**Comment by kelunik** [comment]
Thanks!

**Pull request merged**